autometrics/Cargo.toml

```diff
@@ -82,6 +82,7 @@ axum = { version = "0.6", features = ["tokio"] }
 criterion = "0.5"
 http = "0.2"
 opentelemetry_sdk = "0.19"
+prometheus-client = "0.21"
```

Comment:
this isn't `optional`? you activate it with `dep:prometheus-client` like 60 lines above

Reply:
It needs to be in the dev dependencies because it's used in the examples in the docs, which fail to compile without that being there